spec/controllers/v1/people_controller_spec.rb

```diff
@@ -5,8 +5,8 @@
     it "returns success" do
       user = instance_double("Person", id: 3)
       expect(Person).to receive(:create_or_update).
-        with(email: 'user@example.com', tags: ['test']) { user }
-      post :create, person: { email: 'user@example.com', tags: ['test'] }
+        with(email: 'user@example.com', remote_fields: { tags: ['test'] }) { user }
+      post :create, person: { email: 'user@example.com', remote_fields: { tags: ['test'] } }
```

Comment:
Line is too long. [92/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.